packages/fast-layouts-react/src/grid/grid.props.ts

```diff
@@ -69,6 +74,12 @@ export interface GridHandledProps extends GridManagedClasses {
      * Use this property to support -ms-grid when Grid is a child of an element who's display property is set to -ms-grid
      */
     row?: number;
+
+    /**
+     * The CSS grid display, "grid" when true and "-ms-grid" when false
+     * Provide this prop when doing server side rendering
+     */
+    display?: GridDisplay;
```

Comment:
I'm torn on the naming convention here a bit. `display` initially causes me to think this relates to whether the grid should be displayed or not - though I do understand we have the type system providing context. I do wonder if a modifier may possibly help here? Perhaps something like `displayType` or something which eludes a bit more to it being related to the display property?

Reply:
I can see how display is a bit vague, displayType sounds good! I'll make the change